visualization/Readme.md

```diff
@@ -0,0 +1,33 @@
+Before using this tool, you first need to prepare dataset and install requirements:
+```
+    pip install -r visualization/requirements.txt
+    ./datasets/prepare_all.sh
+```
+
+By default 'math' dataset will be used. You can change it in config.yaml file (dataset: str = "math"), in visualization/settings/visualization_config.yaml or through command line
+
+Command to launch the program (all parameters are optional):
+```
+    python3 visualization/__main__.py \
+    ssh_key=<path_to_ssh> \
+    ssh_server=<server> \
+    hostname=<host> \
```

Comment:
Some details are missing - e.g. do I need to launch server myself before starting this script?